doc/api/crypto.md

```diff
@@ -172,14 +172,14 @@ console.log(encrypted);
 // Prints: ca981be48e90867604588e75d04feabb63cc007a8f8ad89b10616ed84d815504
 ```
 
-### cipher.final([output_encoding])
+### cipher.final([outputEncoding])
```

Comment:
I think the issue is the links out on the internet linking in to particular methods in the node docs. They would then just go to the top of the page.